### PR TITLE
AT_01.05.04 | Login by email tab UI > Insure Password label has text …

### PR DESCRIPTION
**Pull request by shabaiev**
…'Password'

https://trello.com/c/uMyTIAeR/204-at010504-login-by-email-tab-ui-insure-by-email-tab-is-visible

### CODE REVIEW

File: cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.05_LoginByEmailTab_UI.cy.js [hunk 1]
```diff
@@ -25,4 +25,12 @@ describe('US_01.05 | Login By Email Tab UI', () => {
             .should('be.visible')
             .and('have.attr', 'placeholder', this.inputField.loginPopup.emailInputField);
     });
-})
+
+    it('AT_01.05.04 | Login by email tab UI > Insure Password label has text "Password"', () => {
+        loginPopup
+            .getPasswordLabel()
+            .should('be.visible')
+            .and('have.text', 'Password')
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.05_LoginByEmailTab_UI.cy.js:33]: The label text "Password" needs to be moved in the fixtures

File: cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.05_LoginByEmailTab_UI.cy.js [hunk 2]
```diff
@@ -25,4 +25,12 @@ describe('US_01.05 | Login By Email Tab UI', () => {
             .should('be.visible')
             .and('have.attr', 'placeholder', this.inputField.loginPopup.emailInputField);
     });
-})
+
+    it('AT_01.05.04 | Login by email tab UI > Insure Password label has text "Password"', () => {
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.05_LoginByEmailTab_UI.cy.js:29]: change the name of the test. Leave only the name
it('AT_01.05.04 | Insure Password label has text "Password"', ()

Comment (shabaiev) [reply cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.05_LoginByEmailTab_UI.cy.js]: fixed

File: cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.05_LoginByEmailTab_UI.cy.js (hunk 1 above)
Comment (shabaiev) [reply cypress/e2e/testUiAndFunction/startPage/loginPopupSection/US_01.05_LoginByEmailTab_UI.cy.js]: fixed

**Review by Rom-T — APPROVED** [review]

**Comment by shabaiev** [comment]
[AT_01.04.01 > Verify the link Forgot your password? is visible in the footer](https://trello.com/c/2iLXBwdL/278-at010401-verify-the-link-forgot-your-password-is-visible-in-the-footer)

**Comment by shabaiev** [comment]
[AT_01.04.01 > Verify the link Forgot your password? is visible in the footer](https://trello.com/c/2iLXBwdL/278-at010401-verify-the-link-forgot-your-password-is-visible-in-the-footer)

**Pull request merged**